doc/sphinx/language/gallina-extensions.rst

```diff
@@ -2286,31 +2224,27 @@ Multiple binders can be merged using ``,`` as a separator:
 
    Check (forall `{Commutative A, Hnat : !Commutative nat}, True).
 
-One can control the set of generalizable identifiers with
-the ``Generalizable`` vernacular command to avoid unexpected
-generalizations when mistyping identifiers. There are several commands
-that specify which variables should be generalizable.
+.. cmd:: Generalizable {| All Variables | No Variables | {| Variable | Variables } {+ @ident } }
 
-.. cmd:: Generalizable All Variables
+   Controls the set of generalizable identifiers to avoid unexpected generalizations
+   from typos in identifiers.   The effect of multiple :cmd:`Generalizable` commands
+   is cumulative; each command adds additional generalizable :n:`@ident`\s.
```

Comment:
This second sentence is mostly applicable to the "Variable(s)" variants. The "All Variables" and "No Variables" variant completely wipe out the effect of the previous commands.

Reply:
OK.  Reordered the options to make it read better.